teams/4/page-1.css

```diff
@@ -925,6 +929,210 @@ ul li:not(:first-of-type):not(.company__logo) {
   }
 }
 
+/* sliderSection (Nazar) */
+
+.sliderSection {
```

Comment:
for all the classes the proper naming convention is slider-section, avoid using this camel case